src/NUnitFramework/framework/Constraints/CollectionOrderedConstraint.cs

```diff
@@ -37,7 +39,7 @@ public class CollectionOrderedConstraint : CollectionConstraint
     {
         private ComparisonAdapter comparer = ComparisonAdapter.Default;
         private string comparerName;
-        private string propertyName;
+        private Dictionary<string, bool> properties;
         private bool descending;
```

Comment:
This code has not been worked on for a while so it doesn't meet our current standards. All memver fields should get an initial underscore and you should drop `this.` in front of them. However, I suggest saving this for a final pass to make it easier to review the "real" changes.